__tests__/app.test.js

```diff
@@ -331,3 +328,40 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  it("should respond 204 and delete the comment located at given comment_id", () => {
+    const comment_id = 1
+    return request(app)
+    .delete(`/api/comments/${comment_id}`)
+    .expect(204)
+    .then((response) => {
+      expect(response.body).toEqual({})
```

Comment:
just execting the 204 is enough to know the endpoint has worked instead of looking for an empty object.